css-two-column-layout/index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row text-align justify-center-content">
+      <div class="column-full">
+        <header>
+          <div class="column-full">
+            <h1>SCALE, SPEED, AGILITY</h1>
+            <p class="header-margin">We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently and
+            deliver an unparalleled quality of care.</p>
+          </div>
+        </header>
+      </div>
+    </div>
+
+    <div class="row justify-center-content">
+      <div class="column-forth image-align">
+        <img alt="africa-outline" src="images/software.svg">
+      </div>
+      <div class="column-forth add-right-margin">
+        <h3>INTUITIVE, SMART SOFTWARE</h3>
+        <p>We build products that transform how
+          healthcare facilities run across Africa
+          - making them radically more efficient
+          operationally and financially, a complete
+          turnaround.
+        </p>
+      </div>
+      <div class="column-forth image-align">
+        <img alt="stethascope" src="images/practice.svg">
+      </div>
+      <div class="column-forth add-right-margin">
+        <h3>PRACTICE MEDICINE, NOT ADMINISTRATION</h3>
+        <p>Our all-in-one electronic health records
+          (EHR), hospital management (HMIS) and
+          billing system helps streamline your
+          facility, reduce waste and improve
+          quality of care
+        </p>
+      </div>
+    </div>
```

Comment:
Instead of making four `column-fourth` elements, I would make two `column-half` elements, each of which with its own `row` inside of it, that has two `column-half` elements. It is fine to have `row` elements within `column` elements with its own `columns`, and this is an ideal situation for it.